js_kadai2.js

```diff
@@ -0,0 +1,53 @@
+function displayResult(){
+
+    const parentNode = document.getElementById(`display`);
+    //parentNode.innerHTML = `<div id=/"display/"></div>`;//display表示初期化
+    if(parentNode.getElementsByClassName(`children`).length > 0){
+      while (parentNode.firstChild){
+        parentNode.removeChild(parentNode.firstChild);
+      }
+    }
+  
+    const fizz = document.getElementById(`fizz`).value;
+    const buzz = document.getElementById(`buzz`).value;
+  
+    const regex = /^([1-9]\d*)$/;//整数の正規表現
+  
+    if(regex.test(fizz) && regex.test(buzz)){
```

Comment:
今の記述の通り「正しかったら後続の処理をする」という書き方をすると
エラー時の処理がかなり後ろに書かれることになり、見づらいコードになってしまいます。
エラーチェックは以下の記述例の通り「エラー時は即処理を打ち切って後続の処理を行わない」とする方が見やすくなります。

```suggestion
  if (!regex.test(fizz) || !regex.test(buzz)) {
    const pElement = document.createElement(`div`);
    pElement.classList.add("children");
    pElement.textContent = "整数値を入力してください";
    parentNode.appendChild(pElement);
    return;
  }
```